crates/iceberg/src/spec/snapshot.rs

```diff
@@ -122,22 +114,16 @@ impl Snapshot {
     }
     /// Get location of manifest_list file
     #[inline]
-    pub fn manifest_list(&self) -> &ManifestListLocation {
+    pub fn manifest_list(&self) -> &str {
```

Comment:
I think we only need to keep one of these two functions: `manifest_list` or `manifest_list_file_path`.

Reply:
I see and I corrected the duplicated function. Thx.